packages/slate-react/src/components/editable.tsx

```diff
@@ -255,9 +255,7 @@ export const Editable = forwardRef(
               ReactEditor.hasEditableTarget(editor, anchorNode) ||
               ReactEditor.isTargetInsideNonReadonlyVoid(editor, anchorNode)
 
-            const focusNodeSelectable =
-              ReactEditor.hasEditableTarget(editor, focusNode) ||
-              ReactEditor.isTargetInsideNonReadonlyVoid(editor, focusNode)
+            const focusNodeSelectable = ReactEditor.hasTarget(editor, focusNode)
```

Comment:
I'm not sure `focusNodeSelectable` is still the best name for this variable, since the non-void non-editable nodes still aren't selectable. `ReactEditor.toSlateRange` just changes the focus to a node that _is_ selectable now.

Reply:
Also, this PR only fixes non-selectable _focus_ points now, but should non-selectable _anchor_ points be fixed as well? It wouldn't be necessary for my project, but conceivably, if a non-void non-editable node contained non-interactive text, it would be easy to make a browser selection with the anchor inside that node and the focus outside it.

Reply:
Fixing non-selectable anchor points would add extra complexity in that we'd have to decide what to do when a selection starts and ends in the same non-selectable node... This PR is already an improvement on the existing behavior as-is, so maybe fixing anchors can be saved for a future PR? 😅

Reply:
Changed the variable name to `focusNodeInEditor` with https://github.com/ianstormtaylor/slate/pull/5716/commits/98630d11ebf14e40fdcfb59c080530e69c0bfb5e.